requirements.txt

```diff
@@ -7,6 +7,7 @@ django-cors-headers==3.0.2       # allows CORS requests for client-side developm
 elasticsearch==6.4.0             # elasticsearch client
 elasticsearch-dsl==6.4.0	     # elasticsearch query utilities
 gunicorn==19.10.0                # web server
+Jinja2==2.10.1                   # jinja templates are used in the kubernetes deployment scripts
```

Comment:
I'd rather this go in requirements-dev and keep this file for runtime dependencies in seqr, but if you feel strongly we can leave it here